R/modifyInvestmentVariables.R

```diff
@@ -0,0 +1,98 @@
+#' Modify Investment Variables
+#'
+#' Timesteps for investment variables represent a different year range than
```

Comment:
I would reformulate slightly:

Years represented by investment variables in the energy system (vm_deltacap, vm_xxx) {please insert all variables this is applied to} are different than in the normal reporting convention. In the current REMIND version, vm_deltacap(t) represents the average of the years t-4years...t (for 5 year time steps), while in the general reporting convention it represents the average of t-2.5years to t+2.5years.